utils/describe-ref/lib/describe-ref.js

```diff
@@ -56,10 +56,11 @@ function sync(options = {}, includeMergedTags) {
 }
 
 function parse(stdout, options = {}) {
+  const minimalShaRegex = /^([0-9a-f]{7,40})(-dirty)?$/;
```

Comment:
So basically all we're doing is adding a trailing `$` to the combined regex? Huh.